simsogui/Gantt.py

```diff
@@ -63,7 +61,7 @@ def get_selected_items(self):
         for row in range(0, self._list_elements.count()):
             if self._list_elements.item(row).checkState() == Qt.Checked:
                 try:
-                    data = self._list_elements.item(row).data(Qt.UserRole).toPyObject()
+                    data = self._list_elements.item(row).data(Qt.UserRole)
```

Comment:
That try except doesn't make sense now. I don't remember why I did the `.toPyObject`.